hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/functional/ColumnStatIndexTestBase.scala

```diff
@@ -70,7 +69,7 @@ class ColumnStatIndexTestBase extends HoodieSparkClientTestBase {
       .add("c4", TimestampType)
       .add("c5", ShortType)
       .add("c6", DateType)
-      .add("c7", BinaryType)
+      .add("c7", StringType)
```

Comment:
We already have `c2` as StringType, why not jsut comment `c7` with a note for now?

Reply:
I had to change all test fixtures. So, just switch the type.